packages/firestore/src/util/input_validation.ts

```diff
@@ -330,7 +330,7 @@ export function valueDescription(input: AnyJs): string {
     if (input instanceof Array) {
       return 'an array';
     } else {
-      const customObjectName = tryGetCustomObjectType(input);
+      const customObjectName = tryGetCustomObjectType(input!);
```

Comment:
Typescript is not able to narrow it down to exclude null even if we did null check before.